### ISSUE TITLE
how to manually trigger a refresh of the value

### ISSUE THREAD

**Issue by geekflyer** [danielearwicker/computed-async-mobx#13]
Hi,

this an excerpt of my current code:
```ts

export class ConnectorsStore {
  constructor(private _orgStore = orgStore) {}

  promisedConnectors = promisedComputed(null, () => {
    return fetch(`${this._orgStore.selectedOrgId}/connectors/`).catch(e => e);
  });
}
```

and I'm using the `promisedConnectors` property inside a React component. The function correctly refetches the "connectors" when the `selectedOrgId` changes.

However I wonder how I can manually reattempt a fetch / invalidate the cache etc.? 
This is in particular required in cases where the `fetch` request failed / errored and the user wants to retry it (via a "retry" button).
Mobx-utils for example has the `refresh` method on `lazyObservable`: https://github.com/mobxjs/mobx-utils#lazyobservable which serves a similar purpose.

I was able to get a similar behaviour with promisedComputed with the following code, but I feel it's kind of a hack:

```ts
export class ConnectorsStore {
  constructor(private _orgStore = orgStore) {}

  @observable refreshRequests = 0;

  @action.bound
  refresh() {
    this.refreshRequests++;
  }

  promisedConnectors = promisedComputed(null, () => {
    this.refreshRequests;
     return fetch(`${this._orgStore.selectedOrgId}/connectors/`).catch(e => e);
  });
}

```

In my react component I would have a a button `<Button onClick={connectorsStore.refresh}>Retry</Button>`.

Is there a cleaner "refresh" solution with promisedComputed possible?

Thanks and Best Regards Christian

**Comment by danielearwicker** [comment]
Your hack is actually a theoretically correct solution. The `refreshRequests` is a record of how many times the `refresh` method has been called, and that fact is what your compute function depends on.

However as this is very common, it's a great idea to add it to the built-in API.

**Comment by danielearwicker** [comment]
See 3.2.0 and please log any further issues you hit (you are an early adopter of the new API!)

**Comment by geekflyer** [comment]
@danielearwicker  you haven't published `3.2.0` on npm

**Comment by danielearwicker** [comment]
Derp, now I have.
